source/gent/tier1_hortense.rst

```diff
@@ -151,6 +176,28 @@ Project storage
   cd $VSC_SCRATCH_PROJECTS_BASE/<yourprojectnamehere>
 
 
+Read only filesystem
+********************
+
+The performance of the filee system is not always optimal in case of small files often read without
+writing them (software stack, certain input files, for example ones for DNA sequencing). 
```

Comment:
make a separate sentence stating that the centrally installed software is expected to be accessed via the readonly mount